core/src/i18n/en.yaml

```diff
@@ -1000,6 +1001,10 @@ events:
 extensibility:
   configuration-error: 'Configuration error: {{error}}'
   error: Your extension configuration is incorrect.
+  message:
+    old-version: There is a newer version of the extensibility configuration. Please migrate to the newest version.
+    unnown-version: Your extensibility configuration version is no longer supported, but cannot be migrated. Please contact administrator.
```

Comment:
If I remember correctly, we wanted to have a link to the documentation about extensibility versioning here, instead of `Please contact administrator`.